Sources/SpaceEngineers/Content/Data/CubeBlocks.sbc

```diff
@@ -18156,6 +18160,8 @@
       <MaxForceMagnitude>3.36E+07</MaxForceMagnitude>
       <RotorPart>RealWheel1x1</RotorPart>
       <SuspensionLimit>0.6</SuspensionLimit>
+      <MinHeight>-1.5</MinHeight>
+      <MaxHeight>1.3</MaxHeight>
       <PropulsionForce>700</PropulsionForce>
       <DamageEffectId>212</DamageEffectId>
     </Definition>
```

Comment:
I see, indeed, that's either broken or misleading :/ you could report that to github.